gateway/Automation/Automation.hpp

```diff
@@ -82,6 +82,10 @@ void kitchenOff() {
   setOutput(KITCHEN_TABLE_ID, Relay::OFF);
 }
 
+void doorbellclick() {
```

Comment:
Let’s use camelCase naming with callback for callback functions, ie:
‘doorbellClickCallback’